### DISCUSSION TITLE
Username/Password Authentication Failed on Google Colab

### DISCUSSION THREAD

**Discussion by bcreategd** [comfyanonymous/ComfyUI#5740] [Q&A]
When I try to download a resource with this code: !wget -c https://civitai.com/api/download/models/797871 -O ./models/checkpoints/Watercolor_Flux.safetensors 

I get a " Username/Password Authentication". How can I fix this please?

**Comment by ltdrdata** [comment]
Downloading from sites like Civitai, which require download authentication, can be complicated to set up. It is recommended to download the files directly and upload them to a connected Google Drive for use.
